.github/workflows/e2e-ios.yml

```diff
@@ -44,6 +44,12 @@ jobs:
         run: git diff --exit-code
       - name: Check E2E wallet balance
         run: NODE_OPTIONS='--unhandled-rejections=strict' yarn ts-node ./e2e/scripts/check-e2e-wallet-balance.ts
+      - name: Create iOS E2E .env File
+        env:
+          MOCK_PROVIDER_BASE_URL: ${{ secrets.MOCK_PROVIDER_BASE_URL }}
+          MOCK_PROVIDER_API_KEY: ${{ secrets.MOCK_PROVIDER_API_KEY }}
+        working-directory: e2e
+        run: echo MOCK_PROVIDER_BASE_URL=$MOCK_PROVIDER_BASE_URL >> .env && echo MOCK_PROVIDER_API_KEY=$MOCK_PROVIDER_API_KEY >> .env
```

Comment:
what's the purpose of this if the MOCK_PROVIDER config is also set in `e2e/.env.enc`?

Reply:
The github workflow doesn't decrypt the .env.enc file, so the tests will fail in github if this manual .env setup is not included. This is based on an [earlier approach](https://github.com/valora-inc/wallet/pull/534/files#diff-3e103440521ada06efd263ae09b259e5507e4b8f7408308dc227621ad9efa31eR46) to adding config values.

Reply:
Yes that is my understanding.